mautic/mautic
# Fix elfinder js libraries

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

When moving some hard-coded JS libraries to NPM I did not realize that it won't work for production as the node_modules directory won't be part of the prod assets.

So instead I'm loading the libraries.js that already included jquery.js and jquery-ui.js libraries. Although to make it work I had to add 4 new jquery-ui packages to the build.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Run `composer install`
3. Run `bin/console mautic:assets:generate`
4. Disable the GrapeJS plugin
5. Create new email or a landing page
6. In the builder, to to the image library (the ElFinder UI)
7. Test some features here, selecting an image, searching, browsing, whatever you did with it before. It should all work and there should be no error in the browser's dev tools JS console.
